module/mailer/src/Domain/MailMessage.php

```diff
@@ -0,0 +1,142 @@
+<?php
+/**
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Ergonode\Mailer\Domain;
+
+use DateTime;
+use Ergonode\Core\Domain\ValueObject\Language;
+use Ergonode\SharedKernel\Domain\ValueObject\Email;
+
+/**
+ * Abstract class for mail message
+ */
+abstract class MailMessage implements MailMessageInterface
+{
+    /**
+     * @var Email
+     */
+    protected Email $to;
+
+    /**
+     * @var Language
+     */
+    protected Language $language;
+
+    /**
+     * @var Email|null
+     */
+    protected ?Email $from = null;
```

Comment:
One or many? Email supports many from, to, bcc, cc and others. I think i should create EmailCollection object from ArrayCollection and use it. What do U think about it?